src/main/java/io/appform/functionmetrics/FunctionTimerAspect.java

```diff
@@ -90,7 +87,14 @@ private boolean cacheDisabled() {
         return FunctionMetricsManager.getOptions().map(Options::isDisableCacheOptimisation).orElse(false);
     }
 
-    private MethodData createMethodData(Signature callSignature) {
+    private MethodData getMethodData(final ProceedingJoinPoint joinPoint, final Signature callSignature) {
+        return cacheDisabled()
```

Comment:
Change cacheDisabled to return true if param capture is enabled irrespective of the other option

Reply:
I don't understand this. Should cache be automatically disabled if param capture is enabled? This needs to be communicated to the user as well. I suggest doing this in the function metrics manager initialisation then.

Reply:
Got it.